src/main/resources/org/jenkinsci/plugins/urltrigger/URLTrigger/config.jelly

```diff
@@ -61,40 +61,36 @@
                     </f:entry>
 
                     <f:entry>
-                        <table style="width:100%">
-                            <f:checkbox
-                                    field="checkETag"
-                                    name="urlElements.checkETag"
-                                    checked="${urlElements.checkETag}"
-                                    title="${%Check ETag}">
-                            </f:checkbox>
-                        </table>
+                        <f:checkbox
+                                field="checkETag"
+                                name="urlElements.checkETag"
```

Comment:
I've just try to remove names for checkETag and checkLastModificationDate so it works fine on the localhost. As far as I understand we could remove a name attribute for all objects which have a field attribute right?